packages/terra-form-textarea/tests/jest/Textarea.test.jsx

```diff
@@ -12,7 +12,6 @@ it('should render a TextArea when all the possible props are passed into it', ()
   const textarea = (
     <Textarea
       name="description"
-      cols={15}
```

Comment:
why the removal?

Reply:
It's not a supported prop type, I didn't see much value in testing it.. besides specifying columns for a textarea doesn't make much sense.

Reply:
It's actually an html attribute on the dom textarea. https://www.w3schools.com/tags/att_textarea_cols.asp. Does it produce errors if we keep it?